Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline
+{
+    agent
+    {
+        node
+        {
+            label 'embedded-docker'
+        }
+    }
+    environment
+    {
+        MAKEFLAGS = '-j 3'
+        PACKAGE_NAME = 'um_kernel'
+        DOCKER_CONTAINER_NAME = 'um-kernel_build'
+        DOCKER_IMAGE_NAME = 'um-kernel_image'
+        BUILD_OUTPUT_DIR = '_build_armhf'
+    }
+    stages
+    {
+        stage('Prepare')
+        {
+            steps
+            {
+                echo "Starting with a clean environment"
+                step([$class: 'WsCleanup'])
+                checkout scm
+                sh 'git submodule update --init --recursive'
+            }
+        }
+        stage('Build docker image')
+        {
+            steps
+            {
+                echo "Starting build of ${DOCKER_IMAGE_NAME}_${BUILD_NUMBER}"
+                sh 'docker build . -t ${DOCKER_IMAGE_NAME}_${BUILD_NUMBER}'
+            }
+        }
+        stage('Build package')
+        {
+            steps
+            {
+                echo "Starting build of ${PACKAGE_NAME}"
+                sh "mkdir -p ${BUILD_OUTPUT_DIR}"
```

Comment:
i know we use this as our volume mount, i'm not exactly happy with this step yet